### DISCUSSION TITLE
Word games

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#606] [General]
I am not a native English speaker and would like to play some word games in order to extend my vocabulary. Any recommendation ?

**Comment by MirkosMangle** [comment]
I use Duolingo app for learning purposes

**Comment by hernessteve** [comment]
I think Wordle and Scrabble are your best options here.
There is also Jumble, not bad. It might be a little difficult for you at the beginning so make sure you have a word finder on your phone [https://word-finder.com/jumble-solver/](https://word-finder.com/jumble-solver/) and they have a jumble solver as well. This way you can learn and have fun at the same time.

**Reply by ethancarterrr3** [reply]
I agree! Jumble is a fun way to build your word skills, and I’ve also been using a [Jumble Solver](https://jumblesolverss.com/) when I get stuck—it makes learning new words a lot more enjoyable.
